packages/react/src/themes/teams/siteVariables.ts

```diff
@@ -61,6 +61,17 @@ export const lineHeightMedium = 1.4286
 export const lineHeightLarge = 1.3333
 export const lineHeightLarger = 1.3333
 
+//
+// Z-INDEX
+//
+export const zIndexes = {
```

Comment:
Can we expend a bit the description for this object, and add guidance to when the users should use each of this values, like what is the most common usage for them.